docs/sections/calculations/errors.md

```diff
@@ -1,302 +1,68 @@
-Use a combination of the bash command `verdi data structure list` and the python function `load_node()` to get an object representing the structure created earlier.
-
-:::
-
-### Attaching the input information to the calculation
-
-Once you've created a `structure` node and a `kpoints` node, attach it to the calculation:
+You will need to set the rest of the inputs by yourself, the most critical being `structure`, `kpoints`, `pseudos` and the `resources` options under the builder's `metadata`.
+An incomplete example of how to set these parameters are shown below.
 
 ```{code-block} ipython
 
-In [5]: builder.structure = structure
-   ...: builder.kpoints = kpoints
+In [2]: builder.structure = ...
+   ...: builder.kpoints = ...
+   ...: builder.pseudos = ...
+   ...: builder.metadata.options.resources = {'num_machines': 1}
 
 ```
 
-:::{note}
+:::{attention}
 
-The builder accepts both *stored* and *unstored* data nodes.
-AiiDA will take care of storing the unstored nodes upon submission.
-If you decide not to submit, nothing will be stored in the database.
+The `resources` provided above will work for a locally hosted code (which should tipically be the case for these tutorial), but if you are running these tests in a cluster, you may need to set up account permissions or other options (you can consult the section for [schedulers](https://aiida.readthedocs.io/projects/aiida-core/en/latest/topics/schedulers.html) of the AiiDA documentation).
 
 :::
 
-PWscf also needs information on the pseudopotentials, in the form of a dictionary, where keys are the names of the elements and the values are the corresponding `UpfData` objects containing the information on the pseudopotential.
-However, instead of creating the dictionary by hand, we can use a helper function that picks the right pseudopotentials for our structure from a pseudopotential *family*.
-You can list the preconfigured families within the IPython shell using:
-
-```{code-block} ipython
-
-In [6]: !verdi data upf listfamilies
-
-```
-
-Pick the one you pseudos (the `SSSP` family) and link the correct pseudopotentials to the calculation using the command:
-
-```{code-block} ipython
-
-In [7]: from aiida.orm.nodes.data.upf import get_pseudos_from_structure
-   ...: builder.pseudos = get_pseudos_from_structure(structure, '<PSEUDO_FAMILY_NAME>')
-
-```
-
-Print the content of the `pseudos` namespace with `print(builder.pseudos)` to see what the helper function created.
-
-### Preparing and debugging input parameters
+For the `structure` you can download the following {download}`silicon crystal<include/data/Si.cif>` and import it into your database.
+If you have already done so previously (as it is used in other tutorial sections), you may want to use that pre-existing node instead of saving a new node with repeated information.
+To do so you may search for its `PK` by running `verdi data structure list` and then use the function `load_node()` to retrieve it.
```

Comment:
```suggestion
To do so you may search for its PK by running `verdi data structure list` and then use the function `load_node()` to retrieve it.
```

Reply:
I think this is how it was done in my sections, which makes sense, because PK is not a code, but `pk` or `id` is - because it's a class attribute...
Maybe go through the document and see if there are similar corrections (because I'll definitely miss them going through it manually here).